db/seeds.rb

```diff
@@ -4,4 +4,8 @@
 # Examples:
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+#   Character.create(name: 'Luke', movie: movies.first)1)
+
```

Comment:
Small thing, but using the Faker gem in a .times loop would clean this up a little.